VideoLocker/src/main/java/org/edx/mobile/view/LoginActivity.java

```diff
@@ -294,8 +285,7 @@ protected void onOnline() {
     @Override
     protected void onOffline() {
         super.onOffline();
-        showErrorMessage(getString(R.string.no_connectivity),
-                getString(R.string.network_not_connected), false);
+        showErrorMessage(getString(R.string.no_connectivity), getString(R.string.network_not_connected));
```

Comment:
@marcotuts I'd also like to push back on this. 

According to our wiki, which we built by following the Google material guidelines, we should not be creating an alertdialog when the user goes offline, only when a user submits data and see's that the device is offline.

@mdinino 
The current implementation where the alertdialog comes up when the device goes offline does not follow the acceptance criteria:
* Implement "Error Submitting Data" dialog as per https://openedx.atlassian.net/wiki/display/MA/Error+Handling